source/releasenotes/024-05-06-domain-validation.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Domain validation required for all sites "
+published_date: "2024-05-06"
+categories: [security, infrastructure, action-required]
+---
+
+As part of our continued effort to protect sites on our platform, Pantheon now requires all sites on the platform to prove domain ownership via TXT records. This enhances security by ensuring domain ownership and protection, safeguarding against domain hijacking by malicious actors. 
+
+<h3>Highlights</h3>
+
+* Easily validate domain ownership within the Pantheon dashboard.
+* Validated domains are protected from hijacking.
```

Comment:
Is this phrasing accurate? If I'm a regular customer, does validating the domain protect it from hijacking? Or is the requirement to validate a means of ensuring that it is not presently being hijacked?

I think this detail is worth getting straight because I know some customers are perceiving this change as burdensome. Writing this release note as if this change is purely positive might aggravate them more.

Reply:
@conorbauer23 Can you weigh in?

Reply:
@conorbauer23 makes sense. I suggested an edit above that says as much. If you accept that change I think we can remove this `h3` and the bullets underneath it. 

> I believe the phrasing here from a marketing standpoint is true, but maybe we could say it differently?

eh, as a member of the Marketing team I view this change as a common sense security practice. For most customers it's just one more step in the already complicated DNS dance. For some customers who work in large volumes of sites/domains, this change is burdensome. I think all are best served by describing this change as plainly as possible.